src/Pages/Details/Components/Contentlist/Contentlist.js

```diff
@@ -0,0 +1,219 @@
+                  <span className="refund_cost">환불불가</span>
+                </li>
+                <li>
+                  <span className="refund_day">이용 당일</span>
+                  <span className="refund_cost">환불불가</span>
+                </li>
+              </ul>
+              {/* <div className="map_box">
+                <div className="map_text">
+                  <p className="map_title">발리와</p>
+                  <p className="map_adress">
+                    서울 강남구 압구정로10길 30-8 2층 발리와
+                  </p>
+                </div>
+                <div className="map_view"></div>
+              </div> */}
+              <div className="review_container">
+                <div className="review_text">
+                  <h4 id="list5" className="text_title">
+                    이용후기
+                  </h4>
+                  <span>30개</span>
+                  <span className="dot"></span>
+                  <h4 className="text_Average">평균 평점</h4>
+                  <span>4.8</span>
+                </div>
+                <div className="review_box">
+                  <ul className="review_list">
+                    {reviewData &&
+                      reviewData.map(ele => {
+                        return (
+                          <li>
+                            <img
+                              className="user_img"
+                              alt="suhyeon"
+                              src={fixImg}
+                            />
+                            <div className="user_review">
+                              <div className="user_text">
+                                <p className="user_id">{ele.user_nickname}</p>
+                                <p className="user_comment">{ele.content}</p>
+                                <span>{ele.created_at}</span>
+                              </div>
+                              <ul className="like_box">
+                                <li>
+                                  <img alt="like" src="./images/star.svg" />
+                                </li>
+                                <li>
+                                  <img alt="like" src="./images/star.svg" />
+                                </li>
+                                <li>
+                                  <img alt="like" src="./images/star.svg" />
+                                </li>
+                              </ul>
+                            </div>
+                          </li>
+                        );
+                      })}
+                  </ul>
+                  <div className="paging">
+                    <a className="btn_previous" href="/">
+                      <button></button>
+                    </a>
+                    <a href="/">1</a>
+                    <a href="/">2</a>
+                    <a href="/">3</a>
+                    <a href="/">4</a>
+                    <a href="/">5</a>
```

Comment:
₩₩
```suggestion
                    {[...Array(5)].map((_, idx) => {
                      return <a href="/">{idx + 1}</a>;
                    })}
```
이런식으로 구현해줄 수도 있습니다.